src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag

```diff
@@ -64,6 +64,12 @@
                             <f:feedbackSessionPublishButton buttonType="${sessionPanel.feedbackSessionPublishButton.buttonType}"
                                                             publishButton="${sessionPanel.feedbackSessionPublishButton}" />
                         </div>
+                        <br>
+                        <div>
+                            <button id="button_print" onclick="printView();" type="input" class="btn btn-primary btn-block">
```

Comment:
Might be better to avoid the use of `onclick` in newer elements (although I know older code tends to have it this way), try using an event handler instead.